src/modules/app.js

```diff
@@ -0,0 +1,35 @@
+/* eslint no-undefined: "error" */
+
+import ListTemplate from './templateList.js';
+import localGet from './storage.js';
+
+const todoListContainer = document.querySelector('.toDoListItemContainer');
+
+const addList = (description, completed) => {
+  const listAdded = new ListTemplate(description, completed);
+  localGet.push(listAdded);
+  localStorage.setItem('listStorage', JSON.stringify(localGet));
+  window.location.reload();
+};
+
+window.displayList = () => {
+  todoListContainer.innerHTML = '';
+  localGet.forEach((item, id) => {
+    todoListContainer.innerHTML += `
+        <div class='toDoItem'>
+          <input class='item' id='check-${id}', completed)' type='checkbox' value=${item.completed}>
+          <input type='text' class='findInput' id='input-${id}' value=${item.description} />
+          <i onclick='updateList(${id})' class='fa-solid fa-file-pen' id='options-${id}'></i>
```

Comment:
Kindly note that you don't have a function implemented for `updateList` that you are calling here, which is why the list is not being updated and the tasks are not editable. Please fix that 🤗